README.md

```diff
@@ -3,60 +3,80 @@ Administer AACT: Aggregated Analysis of ClinicalTrials.gov
-Then run the generic setup script:
-
-`./bin/setup`
-
-## Environment variables
-
-After running either setup scrupt, you'll have a `.env` file that contains an empty template for the environment variables you'll need. These variables are copied from `.env.example`
-
-Setup the folders you need with `Util::FileManager.new` in the console
-
-setup the databases with
-`RAILS_ENV=test bin/rake db:create`
-`bin/rake db:create`
-`RAILS_ENV=test bin/rake db:migrate`
-`bin/rake db:migrate`
-
-Grant permissions to the read_only role.
-`bin/rake db:setup_read_only`
-`bin/rake db:setup_read_only RAILS_ENV=test`
-
-
-## Sanity checks
-
-Sanity checks are a simple way for us to check that the tables in the database have been imported correctly and gives some insight into how the data looks at a high level. Both the daily and full import rake tasks run the sanity check automatically. To run it manually, open up a Rails console and enter `SanityCheck.run`. This will create a record in the `sanity_checks` table with a report represented in JSON.
-
-## Guidelines
-
-Use the following guides for getting things done, programming well, and
-programming in style.
-
-* [Protocol](http://github.com/thoughtbot/guides/blob/master/protocol)
-* [Best Practices](http://github.com/thoughtbot/guides/blob/master/best-practices)
-* [Style](http://github.com/thoughtbot/guides/blob/master/style)
+You should always setup AACT Core before setting up AACT Admin. AACT Admin relies on AACT Core. So if you haven’t set up AACT Core yet, please do so now. [aact core](https://github.com/ctti-clinicaltrials/aact)  
+Make sure you've set values for the environmental variables 
+- `AACT_DB_SUPER_USERNAME=<superuser_name>`  
+- `AACT_PASSWORD=<superuser_password>`  
+- `AACT_PUBLIC_DATABASE_NAME=aact`  
+- `AACT_PUBLIC_BETA_DATABASE_NAME=aact`  
+- `PUBLIC_DB_USER=<superuser_name>`  
+- `PUBLIC_DB_PASS=<superuser_password>`  
+- `AACT_ALT_PUBLIC_DATABASE_NAME=aact_alt`  
+These variables should have been set when you setup AACT Core. If any are missing you should add them to where you store your variables (for instance ".bash_profile" or ".zshrc").  
+
+Be sure to call `source` on the file where your passwords are. Example `source ~/.bash_profile` so they are reloaded into the terminal.   
+
+You also should have created aact_alt when setting up AACT Core. But if you didn't, you will need to add it now. So enter the psql shell to create it if it had not been created already.  
+
+- `psql postgres -U <super_username>`  
+
+- `\l`  if it's listed, you can exit the shell now. If it is missing, then create it.  
+
+- `template1=# create database aact_alt;`  
+
+- `template1=# \q` (quit out of postgres)  
+
+- Clone the aact-admin repo.  
+
+- `cd` into the aact-admin directory  
+
+- run `bundle install` (bundle version should be '~> 1.17.3')  
+
+if you run into issues with bundle installing `libv8` and the `rubyracer` on a mac you can follow these steps
+`brew install v8-315`  
```

Comment:
Could you add <br> after "these steps"  so it would look like a list?

Reply:
Looks good. Good job.